.golangci.yml

```diff
@@ -0,0 +1,43 @@
+run:
+  timeout: 5m
+  tests: false
+  
+linters:
+  disable-all: true
+  enable:
+    - errcheck
+    - gofmt
+    - golint
+    - govet
+    - ineffassign
+    - misspell
+
+severity:
+  default-severity: error
+
+issues:
+  # use these new lint checks on code since #2574
+  new-from-rev: 988ad6384783701d0b78bed903ff1bc28277f05c
```

Comment:
Is it a lot of work to make indexer clean for the error-level linters you've picked out? just curious.

Reply:
if we start from this commit, there are just a couple of errors.